main.py

```diff
@@ -2,6 +2,43 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    actors = [
+        ("George", "Klooney"),
```

Comment:
The actor's name 'George Klooney' should be corrected to 'George Clooney' before creating the object, or ensure the update operation is intended to correct this typo.